recipes/bwa-mem2/build.sh

```
@@ -9,7 +9,13 @@
     sed -i.bak 's/memset_s/memset8_s/g' ext/safestringlib/safeclib/memset_s.c
     sed -i.bak 's/memset_s/memset8_s/g' ext/safestringlib/safeclib/wmemset_s.c
 fi
-LIBS="${LDFLAGS}" make CC="${CC}" CXX="${CXX}" multi
+
+case ${ARCH} in
+  x86_64) LIBS=“${LDFLAGS}” make CC=“${CC}” CXX=“${CXX}” multi ;;
```

Shellcheck

[warning] This is a unicode quote. Delete and retype it (or quote to make literal).

(SC1110)